packages/extension/src/newtab/DndModal.tsx

```diff
@@ -113,36 +46,70 @@ export default function DndModal({
     return `daily.dev in a new tab is paused, it will resume on ${date} at ${time}`;
   };
 
-  const titleSize = isActive ? 'typo-title3' : 'typo-title2';
-
   return (
-    <StyledModal
+    <Modal
+      kind={Modal.Kind.FlexibleTop}
+      size={Modal.Size.Medium}
       {...modalProps}
       onRequestClose={onRequestClose}
-      style={{
-        content: {
-          paddingTop: '2rem',
-          maxWidth: '27.5rem',
-          display: 'flex',
-          flexDirection: 'column',
-        },
-      }}
     >
-      {!isActive && <ModalCloseButton onClick={onRequestClose} />}
-      <div className="px-16 text-center">
-        <h3
-          className={classnames(
-            'font-bold text-theme-label-primary',
-            titleSize,
-          )}
-        >
-          Pause new tab
-        </h3>
-        <p className="mt-1 typo-callout text-theme-label-secondary">
+      <Modal.Header title="Pause new tab" />
+      <Modal.Body>
+        <p className="typo-callout text-theme-label-secondary">
           {getDescription()}
         </p>
-      </div>
-      {isActive ? turnedOnContent : turnedOffContent}
-    </StyledModal>
+        {!isActive ? (
```

Comment:
```suggestion
        {!isActive && (
```

Reply:
I recently found myself avoiding these for best practice as they can introduce null of false renders.
We can make a global web decision about it though.

As for reference:
https://twitter.com/TkDodo/status/1592276656261140480?s=20&t=pmNq6lGDpYM396S-amgkDw

Reply:
You mean the case for numbers?

something like this?
`items.length && <Component />`

For that I always used (and we enforced in my prev company) to cast to boolean first.

`!!items.length && <Component />`

Reply:
Hmm yes that the main concern, but apparently it can also happen with null values (useState) 
It's the safer way to be explicit, not sure how you feel about that?

I changed it to `&&` for now, but perhaps something to consider in the future, as explicit null is always safer.